client/src/components/DragAndDrop/reorder.jsx

```diff
@@ -0,0 +1,35 @@
+
+// REORDERING THE RESULT
+const reorder = (list, startIndex, endIndex) => {
+    const result = Array.from(list);
+    const [removed] = result.splice(startIndex, 1);
+    result.splice(endIndex, 0, removed);
+    return result;
+};
```

Comment:
This function seems to be for moving an element from one index to another. We can probably choose more descriptive names for the function/parameters. If we can't come up with anything, we can expand the comment a little bit (as is, it's redundant as the name `reorder` already implies this function reorders something 👍🏼 ).